pkg/kubernetes/label.go

```diff
@@ -10,6 +10,7 @@ const (
 	// https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
 	LabelKubernetesName      = `app.kubernetes.io/name`
 	LabelKubernetesManagedBy = `app.kubernetes.io/managed-by`
+	LabelSpinnakerSequence   = `moniker.spinnaker.io/sequence`
```

Comment:
Change this to `LabelSpinnakerMonikerSequence` to match the annotation.

Reply:
done